src/vm/eventtrace.cpp

```diff
@@ -6760,9 +6762,9 @@ VOID ETW::MethodLog::SendHelperEvent(ULONGLONG ullHelperStartAddress, ULONG ulHe
                                      ulHelperSize, 
                                      0, 
                                      methodFlags, 
-                                     NULL, 
+                                     W16_NULLSTRING,
```

Comment:
i am not entirely sure if this is necessary will leave it to @brianrob to make the call